aws_lambda_builders/workflow.py

```diff
@@ -138,6 +156,19 @@ def is_supported(self):
 
         return True
 
+    def get_executable(self):
+        """
+        Non specialized path resolver that just returns the first executable for the runtime on the path.
+        """
+        return PathResolver(runtime=self.runtime).path
```

Comment:
Should we make the attribute `path` more explicit? Something like `exec_path`?

Should we force `get_executable` into a strong contract by returning an object that has specific required methods or attributes, instead of just a string?

Reply:
what are other attributes did you have in mind?

Reply:
I didn't have any directly in mind but everywhere this method is doing some `PathResolver().path`. Instead, we can make `get_executable()` return the PathResolver and then when needed, we call the attribute or methods we need. This is what we are doing for the validation.